api/src/controllers/team.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * This module contains handlers for the "team" route.
+ */
+const axios = require("axios");
+
+const mongoose = require("mongoose");
+
+const Team = require("../models/team.model");
+
+
+const options = {
+    headers: {'Authorization': "Bearer " + process.env.TWITTER_BEARER_TOKEN}
+};
+
+/**
+ * Associates a twitter handle with a team on the /team/twitter-handle/:team-id endpoint. 
+ * @param {*} req request object, containing the team_id in the url and twitter handle in the body
+ * @param {*} res response object
+ * @returns 200: successful added twitter handle to team
+ * @returns 400: team id is not in a valid hexadecimal format
+ * @returns 404: team is not found, or handle is invalid
+ * @returns 500: error trying to update the document in db
+ */
+async function storeHandle(req, res) {
+    const {team_id: team_id} = req.params;
+    const {twitterHandle: handle} = req.body;
+
+    // validate if the team exists
+    if (mongoose.Types.ObjectId.isValid(team_id)) {
+        var foundTeam = await Team.findById(team_id);
```

Comment:
minor: prefer `let` over `var`
see: https://www.geeksforgeeks.org/difference-between-var-and-let-in-javascript/#:~:text=var%20and%20let%20are%20both,program%20as%20compared%20to%20let.